java/client/src/main/java/glide/api/commands/ServerManagementClusterCommands.java

```diff
@@ -0,0 +1,66 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.commands;
+
+import glide.api.models.ClusterValue;
+import glide.api.models.commands.InfoOptions;
+import glide.api.models.configuration.RequestRoutingConfiguration.Route;
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * Server Management Commands interface.
+ *
+ * @see <a href="https://redis.io/commands/?group=server">Server Management Commands</a>
+ */
+public interface ServerManagementClusterCommands {
+
+    /**
+     * Get information and statistics about the Redis server. DEFAULT option is assumed. The command
+     * will be routed to all primaries.
+     *
+     * @see <a href="https://redis.io/commands/info/">redis.io</a> for details. {@link
+     *     InfoOptions.Section#DEFAULT} option is assumed.
+     * @return Response from Redis cluster with a <code>String</code> containing the information for
```

Comment:
Without routing info the request is routed to multi-nodes, so the return value is a map of strings. should be

Reply:
Can you add an example, so it would be clearer that getMulti needs to be called in these functions

Reply:
How is this: 
```    
    /**
     * Get information and statistics about the Redis server. DEFAULT option is assumed. The command
     * will be routed to all primaries.
     *
     * @see <a href="https://redis.io/commands/info/">redis.io</a> for details. {@link
     *     InfoOptions.Section#DEFAULT} option is assumed.
     * @return Response from Redis cluster with a <code>Map{@literal <String, String>}</code> with each address
     *     as the key and its corresponding value is the information for the node.
     * @example
     *     <p><code>
     *     {@literal Map<String, String>} routedInfoResult = clusterClient.info().get().getMultiValue();
     *     </code>
     */
```